[*] Running ...
[*] Dataset loaded: C:\Users\Nam\Documents\GitHub\HSEvo/problems/cvrp_aco\dataset/train50_dataset.npy with 10 instances.
C:\Users\Nam\Documents\GitHub\HSEvo\problems\cvrp_aco\gpt.py:22: RuntimeWarning: overflow encountered in exp
  if not feasible_nodes:
C:\Users\Nam\Documents\GitHub\HSEvo\problems\cvrp_aco\gpt.py:33: RuntimeWarning: invalid value encountered in divide
  current_capacity -= demands[next_node]
Traceback (most recent call last):
  File "C:\Users\Nam\Documents\GitHub\HSEvo/problems/cvrp_aco/eval.py", line 60, in <module>
    obj = solve(node_pos, demand)
  File "C:\Users\Nam\Documents\GitHub\HSEvo/problems/cvrp_aco/eval.py", line 28, in solve
    heu = heuristics(dist_mat.copy(), node_pos.copy(), demand.copy(), CAPACITY) + 1e-9
          ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Nam\Documents\GitHub\HSEvo\problems\cvrp_aco\gpt.py", line 34, in heuristics_v2
    current_node = next_node
              ^^^^^^^^^^^^^^
  File "C:\Python313\Lib\random.py", line 489, in choices
    raise ValueError('Total of weights must be finite')
ValueError: Total of weights must be finite
